FernFlower-Patches/0041-More-correctly-handle-inner-class-synthetic-paramete.patch

```diff
@@ -0,0 +1,155 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: zml <zml@stellardrift.ca>
+Date: Thu, 13 May 2021 23:55:33 -0700
+Subject: [PATCH] More correctly handle inner class synthetic parameters
+
+
+diff --git a/src/org/jetbrains/java/decompiler/main/rels/NestedClassProcessor.java b/src/org/jetbrains/java/decompiler/main/rels/NestedClassProcessor.java
+index fe372662be3fbd310f94831b47f21b92033a851c..f50e767ab81de739cad2059b5a8279fcd7cdba25 100644
+--- a/src/org/jetbrains/java/decompiler/main/rels/NestedClassProcessor.java
++++ b/src/org/jetbrains/java/decompiler/main/rels/NestedClassProcessor.java
+@@ -35,6 +35,8 @@ import java.util.*;
```

Comment:
I've tested this and it does not seem to effect decompiled code at all. So i'm gunna split this out and we can discuss it in morre depth later.